doc/api/events.md

```diff
@@ -935,13 +935,415 @@ if the `EventEmitter` emits `'error'`. It removes all listeners when
+target.addEventListener('foo', handler4, { once: true });
+
+target.dispatchEvent({ type: 'foo' });
+```
+
+When the `eventTarget.emit()` function is used. the arguments passed
+following the `type` are composed into a single Array object with an
+additional `type` property:
+
+```js
+const target = new EventTarget();
+target.on('foo', (event) => {
+  console.log(event.type);  // Prints 'foo'
+  console.log(event[0]);  // Prints 'a'
+});
+target.emit('foo', 'a');
+```
+
+### `EventTarget` Error Handling
+
+When a registered event listener throws (or returns a Promise that rejects),
+by default the error will be forwarded to the `process.on('uncaughtException`)
+event. Throwing within an event listener will *not* stop the other registered
+handlers from being invoked.
+
+Similarly to `EventEmitter`, the `EventTarget.captureRejectionSymbol` property
+may be set on the `EventTarget` to override the default error handling and
+prevent the `'uncaughtException'` event from being emitted.
+
+```js
+const target = new EventTarget();
+target[EventTarget.captureRejectionSymbol] = (error, event) => {
+  console.log(error.message, event.type);
+};
+```
+
+This error handling behavior means that errors thrown within an event
+listener *do not terminate the Node.js process by default*.
+
+The `EventTarget` does not implement any special default handling for
+`'error'` type events.
+
+### Class: `Event`
+<!-- YAML
+added: REPLACEME
+-->
+
+The `Event` object is an adaptation of the [`Event` Web API][].
+
+#### Constructor: `new Event(type[, options])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `type` {string}
+* `options` {Object}
+  * `cancelable` {boolean}
+
+#### `event.cancelable`
+<!-- YAML
+added: REPLACEME
+-->
+
+* Type: {boolean}
+
+#### `event.defaultPrevented`
+<!-- YAML
+added: REPLACEME
+-->
+
+* Type: {boolean}
+
+Will be `true` if `cancelable` is `true` and `event.preventDefault()` has been
+called.
+
+#### `event.preventDefault()`
+<!-- YAML
+added: REPLACEME
+-->
+
+Sets the `defaultPrevented` property to `true` if `cancelable` is `true`.
+
+#### `event.timestamp`
+<!-- YAML
+added: REPLACEME
+-->
+
+* Type: {number}
+
+The millisecond timestamp when the `Event` object was created.
+
+#### `event.type`
+<!-- YAML
+added: REPLACEME
+-->
+
+* Type: {string}
+
+The event type identifier.
+
+### Class: `EventTarget`
+<!-- YAML
+added: REPLACEME
+-->
+
+#### Constructor: `new EventTarget()`
+<!-- YAML
+added: REPLACEME
+-->
+
+Creates a new `EventTarget` object.
+
+#### `eventTarget.addEventListener(type, listener[, options])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `type` {string}
+* `listener` {Function|EventListener}
+* `options` {Object}
+  * `once` {boolean}
+
+Adds a new handler for the `type` event. Any given `listener` will be added
+only once per `type`.
+
+If the `once` option is `true`, the `listener` will be removed after the
+next time a `type` event is dispatched.
+
+#### `eventTarget.addListener(type, listener[, options])`
```

Comment:
EventEmitter and EventTarget have an unfortunate naming collision here... `addListener` seems like it could cause confusion because there's nothing in the name that tells how it's different from `addEventListener`. It's obvious if you are aware of the context of pre-existing EventEmitter and EventTarget APIs and the differences between them, but if you aren't, it seems like it would be a bit mystifying.

That said, I'm not sure if there's anything that can be done about that. Although I don't think it's all that necessary to provide EventEmitter convenience APIs on Node's EventTarget.

Reply:
Given that both of these are functions from existing APIs, there's really nothing we can do to rename them to something less confusing.